common/src/main/scala/uk/ac/wellcome/models/Transformable.scala

```diff
@@ -7,18 +7,37 @@ trait Transformable {
   def transform: Try[Work]
 }
 
-case class MiroTransformableData(image_title: Option[String])
+case class MiroTransformableData(
+  image_title: Option[String],
+  image_image_desc: Option[String],
+  image_supp_lettering: Option[String]
+)
 
 case class MiroTransformable(MiroID: String,
                              MiroCollection: String,
                              data: String)
     extends Transformable {
   override def transform: Try[Work] =
     JsonUtil.fromJson[MiroTransformableData](data).map { miroData =>
-      Work(identifiers =
-                    List(SourceIdentifier("Miro", "MiroID", MiroID)),
-                  label = miroData.image_title.getOrElse("no label found"))
+      MiroCollection match {
+        case "Images-A" => transformImagesA(miroData)
+        case _ => throw new Exception(s"Unable to transform unknown collection $MiroCollection")
+      }
     }
+
+  // The mapping of Miro fields to UnifiedItem fields is quite rough,
+  // and based on a cursory expection of the Miro data.
+  // TODO: Get a proper mapping of fields.
+
+  private def transformImagesA(miroData: MiroTransformableData): UnifiedItem =
+    UnifiedItem(
+      identifiers = List(SourceIdentifier("Miro", "MiroID", MiroID)),
+      label = miroData.image_title.get,
+      description = miroData.image_image_desc,
+      lettering = miroData.image_supp_lettering,
+      hasCreatedDate = None,
+      hasCreator = List()
```

Comment:
If these (hasCreatedDate, hasCreator ) can be easily filled out - i'd just do it as a string right now.